airflow/config_templates/default_airflow.cfg

```diff
@@ -377,6 +377,9 @@ update_fab_perms = True
 # 0 means never get forcibly logged out
 force_log_out_after = 0
 
+# 'Recent Tasks' stats will show for old DagRuns if True
```

Comment:
Hi. Can you add some documentation?  We do not yet have a document describing the configuration options of the webserver. Would you like to create it? If not, that's fine and you can also create a new document in the howto catalog.

Reply:
see https://github.com/apache/airflow/pull/7037